specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-03-01/dedicatedHost.json

```diff
@@ -1025,10 +1090,15 @@
       "description": "Specifies information about the Dedicated host."
     },
     "DedicatedHostUpdate": {
+      "type": "object",
       "properties": {
         "properties": {
           "x-ms-client-flatten": true,
           "$ref": "#/definitions/DedicatedHostProperties"
+        },
+        "sku": {
+          "$ref": "./computeRPCommon.json#/definitions/Sku",
+          "description": "SKU of the dedicated host that DedicatedHost can be resized to. Only name is required to be set. Resizing can be only used to scale up DedicatedHost. ListAvailableSizes for a list of possible values."
```

Comment:
[List all available dedicated host sizes for resizing](https://docs.microsoft.com/rest/api/compute/dedicated-hosts/listavailablesizes)